use-cases/manual-org-three-way.md

```diff
@@ -243,66 +247,82 @@ compute; work with the SQS and access S3 resources (this last one via assumeRole
 }
 ```
 
-Now we will deploy the compute component, depending on the compute service
+### Deploy Cloud Connector
 
-#### A) EKS
+#### EKS
 
 
 <!--
+
 1. Kubernetes **Credentials** creation
    - This step is not really required if Kubernetes role binding is properly configured for the deployment, with an
      IAM role with required permissions listed in following points.
    - Otherwise, we will create an AWS user `SYSDIG_K8S_USER_ARN`, with `SYSDIG_K8S_ACCESS_KEY_ID` and
      `SYSDIG_K8S_SECRET_ACCESS_KEY`, in order to give Kubernetes compute permissions to be able to handle S3 and SQS operations
    - Secure for Cloud [does not manage IAM key-rotation, but find some suggestions to rotate access-key](https://github.com/sysdiglabs/terraform-aws-secure-for-cloud/tree/master/modules/infrastructure/permissions/iam-user#access-key-rotation)<br/><br/>
--->
+     -->
 
+1. Retrieve your `<SYSDIG_SECURE_ENDPOINT>` and `<SYSDIG_SECURE_API_TOKEN>`.
 
-If using Kubernetes, we will make use of the [Sysdig cloud-connector helm chart](https://charts.sysdig.com/charts/cloud-connector/) component.
-<br/>Locate your `<SYSDIG_SECURE_ENDPOINT>` and `<SYSDIG_SECURE_API_TOKEN>`.<br/> [Howto fetch ApiToken](https://docs.sysdig.com/en/docs/administration/administration-settings/user-profile-and-password/retrieve-the-sysdig-api-token)
-
-Provided the following `values.yaml` template
-```yaml
-sysdig:
-  url: "https://secure.sysdig.com"
-  secureAPIToken: "SYSDIG_API_TOKEN"
-telemetryDeploymentMethod: "helm_aws_k8s_org"		# not required but would help us
-aws:
-    region: <SQS-AWS-REGION>
-ingestors:
-    - cloudtrail-sns-sqs:
-        queueURL:"<URL_CLOUDTRAIL_SNS_SQS>"             # step 3
-        assumeRole:"<ARN_ROLE_SYSDIG_S3_ACCESS>"        # step 4
-```
+   See [Retrieve API Token](https://docs.sysdig.com/en/docs/administration/administration-settings/user-profile-and-password/retrieve-the-sysdig-api-token) for more information.
 
-We will install it
-```shell
-$ helm upgrade --install --create-namespace -n sysdig-cloud-connector sysdig-cloud-connector sysdig/cloud-connector -f values.yaml
-```
+2. Use the [Sysdig cloud-connector helm chart](https://charts.sysdig.com/charts/cloud-connector/) to Locate your `<SYSDIG_SECURE_ENDPOINT>` and `<SYSDIG_SECURE_API_TOKEN>`.
 
-Test it
-```shell
-$ kubectl logs -f -n sysdig-cloud-connector deployment/sysdig-cloud-connector
-```
+   Use the following `values.yaml` template:
 
-And if desired uninstall it
-```shell
-$ helm uninstall -n sysdig-cloud-connector sysdig-cloud-connector
-```
+   ```yaml
+   sysdig:
+     url: "https://secure.sysdig.com"
+     secureAPIToken: "SYSDIG_API_TOKEN"
+   telemetryDeploymentMethod: "helm_aws_k8s_org"		# not required but would help us
+   aws:
+       region: <SQS-AWS-REGION>
+   ingestors:
+       - cloudtrail-sns-sqs:
+           queueURL:"<URL_CLOUDTRAIL_SNS_SQS>"             # step 3
+           assumeRole:"<ARN_ROLE_SYSDIG_S3_ACCESS>"        # step 4
+   ```
+
```

Comment:
I would add:

For additional information on authentication options, check  `serviceAccount.*` and `aws.*` values